src/screens/Customers/CustomerProfile.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import BaseHeader from '../../components/BaseComponents/BaseHeader';
+import OutlinedColCard from '../../components/OutlinedCardList';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import AddIcon from '@material-ui/icons/Add';
+import ListAltOutlinedIcon from '@material-ui/icons/ListAltOutlined';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+import { CustomerRecord } from '../../utils/airtable/interface';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+    },
+    content: {
+      margin: '0 25px',
+      textAlign: 'left',
+      color: theme.palette.text.primary,
+    },
+    header: {
+      marginTop: '15px',
+    },
+    button: {
+      borderRadius: '12px',
+      height: '30px',
+      color: 'white',
+      backgroundColor: theme.palette.primary.main,
+    },
+    buttonSecondary: {
+      borderRadius: '12px',
+      backgroundColor: theme.palette.secondary.main,
+    },
+  });
+
+interface CustomerProps {
+  classes: any;
+  location: any;
```

Comment:
same comments as before about classes/match/location props

Reply:
- resolved for classes and match
- struggling to correctly type location, but since we're probably going to eventually replace this w redux store anyways, don't think it's worth digging deeper (unless you think it's important)

Reply:
Not very important; thanks for making the initial change though!